ui_curses.c

```diff
@@ -2329,7 +2333,9 @@ static void init_curses(void)
 static void notify_init(void)
 {
 	int fildes[2];
-	pipe(fildes);
+	if (pipe(fildes) == -1) {
+		d_print("pipe failed: %s\n", strerror(errno));
```

Comment:
if `notify_init()` fails, calls to `ui_curses_notify()` will also fail. This means that there will be an inconsistent state. I think its better to exit cmus with an error message at this point.